src/libs/live_view.c

```diff
@@ -524,13 +524,7 @@ void gui_post_expose(dt_lib_module_t *self, cairo_t *cr, int32_t width, int32_t
     // if the user points at this image, we really want it:
     if(!img) img = dt_image_cache_get(darktable.image_cache, imgid, 'r');
 
-    int zoom = 1;
-    float imgwd = 0.90f;
-    if(zoom == 1)
-    {
-      imgwd = .97f;
-    }
-
+    float imgwd = 0.97f;
```

Comment:
Indeed, looks like a code after refactoring :)

Reply:
And as we are at it, make imgwd a const to make it clear that it does not change.